ls1intum/Artemis
# [Enhancement - UI] Exercise-update components checkbox size adjustment

### Checklist

### Motivation and Context
Checkboxes (Options) had different size. This PR unifies the size and adjusts them properly.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration (course should have presentation score enabled)
3. Navigate to Exercises
4. Create/Update an exercise
5. Look at checkboxes

### Screenshots
#### Before:
<img width="552" alt="Screenshot 2021-08-07 at 19 00 43" src="https://user-images.githubusercontent.com/51077603/128608134-6927819f-9b36-450c-974b-609a9332c26a.png">

#### After:
<img width="511" alt="Screenshot 2021-08-07 at 18 45 43" src="https://user-images.githubusercontent.com/51077603/128608143-73f35680-69db-44df-8b7c-cac31c472379.png">
